server/src/controllers/gmail.js

```diff
@@ -112,6 +149,102 @@ class Gmail {
     }
   };
 
+  // Gmail notifications webhook action
+  static emailResponse = async (req, res) => {
+    let buffer = new Buffer(req.body.message.data, "base64");
+    let messageData = JSON.parse(buffer.toString("utf8"));
```

Comment:
we should be using const here... always favor const over let

Reply:
Updated to use const